### PR TITLE
updated to contain proposed endpoints based on GCB feedback

### PR DESCRIPTION
**Pull request by dmlond**

### CODE REVIEW

File: api_design/DDS-1061-optimized_directory_structure_endpoint.md [hunk 1]
```diff
@@ -12,143 +12,67 @@ The goal of this development epic is to produce a set of endpoints to be used in
...
+We started with an exemplar of the response provided by the current children endpoint. The goal of this phase was to tease out the specific information in this response, and craft a new JSON structure with these specific pieces of information only. This was used to create a proposed Summary of impacted APIs and API Specification (below).
+
+## Implementation View
+
+#### Summary of impacted APIs
+
+|Endpoint |Description |
+|---|---|
+| `GET /projects/:id/files` | Get list of files in a project. |
+| `GET /folders/:id/files` | Get list of files in a folder. |
```
Comment (dmann) [inline api_design/DDS-1061-optimized_directory_structure_endpoint.md]: Let's focus on `GET /projects/:id/files` and lose the `GET /folders/:id/files` for now.

File: api_design/DDS-1061-optimized_directory_structure_endpoint.md [hunk 2]
```diff
@@ -12,143 +12,67 @@ The goal of this development epic is to produce a set of endpoints to be used in
...
+
+#### API Specification
+
+##### List project / folder files
+
+`GET /projects/{id}/files` / `GET /folders/{id}/files`
```
Comment (dmann) [inline api_design/DDS-1061-optimized_directory_structure_endpoint.md]: Please remove '/ `GET /folders/{id}/files`

**Review by dmann — CHANGES_REQUESTED** [review]
Two changes, the rest looks good. 👍

**Review by Donmurry — APPROVED** [review]
👍

**Comment by dmlond** [comment]
removed references to `GET /folders/:id/files`

**Review by dmann — APPROVED** [review]

**Pull request merged**